EIPS/eip-X-ASIC-resistant-block-mining.md

```diff
@@ -0,0 +1,182 @@
+---
+eip: <to be assigned>
+title: Poisoning the Well
+author: David Stanfill <david@airsquirrels.com> 
+discussions-to: <email address>
+status: Draft
+type: Standards Track
+category Core
+created: 2018-04-03
+---
+
+
+## Simple Summary
+
+This EIP attempts to break ASIC miners specialized for the current ethash
+mining algorithm.
+
+
+## Abstract
+
+There are companies who currently have dedicated hardware based ethereum miners in
+production, and may be actively mining.  This EIP aims to "poison
+the well" by modifying the block mining algorithm in a low risk manner that
+may *"break"* these miners if they are in-fact built as traditional ASICs.
+
+
+## Motivation
+
+ASIC based miners will have lower operational costs than GPU based miners which
+will result in GPU based mining quickly becoming unprofitable.  Given that
+production of ASIC based miners has a high barrier to entry and few market players,
+this will cause a trend towards centralization of mining power.  
```

Comment:
I _think_ the argument trying to be made here is along the lines of:
> ASIC miner production is likely to be done by the people doing the mining, unlike GPU production which is done primarily by resellers who do not use the GPUs (in mass quantities) themselves.

I would like to see some data to back this up if it is true.  If ASICs are produced and _sold_ to the general public then ASICs are not a problem as it doesn't lead to centralization.  If ASICs are produced and then used (without being sold publicly) then ASICs can lead to centralization if they are a big enough improvement over GPUs (which are sold publicly).

Reply:
Many risk arguments exist around single supplier scenarios as well. Backdoors are possible, leading to future 51% attack risks. 

The data is a difficult ask, given that Bitmain mines in secret there is no way to know for certain. The only probable data is that the miner exists. The closest to data we could get would be from the results of the hardfork. I might suggest watching Monero’s hashrate this weekend as a possible canary in the coal mine.